internal/transport/controlbuf.go

```diff
@@ -416,6 +416,9 @@ func (c *controlBuffer) get(block bool) (interface{}, error) {
 		select {
 		case <-c.ch:
 		case <-c.done:
+			if logger.V(logLevel) {
+				logger.Infof("transport: trigger loopy to exit (with a subsequent conn closure) because transport's context expired.")
```

Comment:
Wondering if this is appropriate at `INFO` level or is `WARNING` more appropriate?

See: https://github.com/grpc/grpc-go/blob/master/Documentation/log_levels.md

Same for other logs in this file/PR.

Reply:
For this case, ErrConnClosing was being returned to the run() goroutine of loopy, which would trigger the exit with an info log. However, I am changing this to plumb data into ErrConnClosing (or a certain type), and will log the error message plumbed in (i.e. I will put the message from this into the error) because ErrConnClosing has nothing in it currently. See https://github.com/grpc/grpc-go/blob/master/internal/transport/controlbuf.go#L536 for how it currently works on master.